### PR TITLE
Notify sound inject

### PR DESCRIPTION
**Pull request by vladest**
Test plan
- Open Tablet
- Press Snap button.
- Make sure there is shutter sound produced
- Switch audio output which is able to produce sound. 
- Repeat Snap
- Make sure shutter sound produced on new audio output

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 06406ef7de3ec33cfbcf0d4e0f2b141c7ed8a4f8 available at: https://gist.github.com/78287bce4f6a7c43e01efc114bd3d1a7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-06406ef.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-06406ef.dmg

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -1785,6 +1789,8 @@ void Application::cleanupBeforeQuit() {
     DependencyManager::destroy<AudioClient>();
     DependencyManager::destroy<AudioInjectorManager>();
 
+//    delete _snapshotSound;
+//    _snapshotSound = nullptr;
```
Comment (kencooke) [inline interface/src/Application.cpp]: Should probably clean up with:
```
    if (_snapshotSoundInjector) {
         // stop will cause the AudioInjector to delete itself.
        _snapshotSoundInjector->stop();
    }
```

File: interface/src/Application.h [hunk 2]
```diff
@@ -682,6 +684,8 @@ private slots:
     QTimer _addAssetToWorldErrorTimer;
 
     FileScriptingInterface* _fileDownload;
+    AudioInjector* _snapshotSoundInjector{ nullptr };
```
Comment (kencooke) [inline interface/src/Application.h]: formatting nitpick.. should be a space before { nullptr }

**Comment by hifi-gustavo** [comment]
Build console output for commit: b2533b6106c12a9f587833a761cc408fb854892c available at: https://gist.github.com/78287bce4f6a7c43e01efc114bd3d1a7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-b2533b6.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-b2533b6.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2665ede06fd8ae665af79b65e3d96a64559ce6dd available at: https://gist.github.com/78287bce4f6a7c43e01efc114bd3d1a7<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-2665ede.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-2665ede.dmg

**Comment by claudioshg** [comment]
testing

**Comment by kencooke** [comment]
### QA results
verified "snap.wav" loaded via hifi.log (**SUCCESS**)
verified snapshot sound is played on default audio device (**SUCCESS**)
verified snapshot sound is played on alternate audio device (**SUCCESS**)

waiting for @claudioshg results

**Comment by claudioshg** [comment]
### Desktop PC - Windows 10 Home - CPU 4.0GHz Intel i7 - GTX1070 - RAM 16GB - Tested with Oculus Rift

Tested: Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10248-2665ede.exe

PASS:
- Shutter sound did produce via ( PC Speakers ) and ( Oculus Rift Audio ).
- Followed all steps in Test Plan, and no problems emerged during testing.

**Pull request merged**